manifests/init.pp

```diff
@@ -173,6 +177,7 @@
   $python_loglevel                                            = $::puppetboard::params::python_loglevel,
   $python_proxy                                               = $::puppetboard::params::python_proxy,
   $python_index                                               = $::puppetboard::params::python_index,
+  Variant[Boolean, Undef] $python_use_epel                    = undef,
```

Comment:
It's probably more common to use `Optional[Boolean] $var_name = undef` but I think that's just a style thing.